pages.de/common/git-send-email.md

```diff
@@ -0,0 +1,24 @@
+# git send-email
+
+> Sende Patches als Email. Möglich für einzelene Commits, Ordner oder .patch Datein.
+> Weitere Informationen: <https://git-scm.com/docs/git-send-email>.
+
+- Sende den letzten Commit der aktuellen Branch:
+
+`git send-email -1`
+
+- Sende einen spezifischen Commmit:
+
+`git send-email -1 {{commit}}`
+
+- Sende die letzten (z.B. 10) Commits der aktuellen Branch:
+
+`git send-email {{-10}}`
+
+- Sende eine Einleitungs Email für eine Reihe von Patches:
```

Comment:
```suggestion
- Editiere eine E-Mail mit einer Reihe von Patches im Standardmailclienten:
```

English page is not precise for this example, see https://git-scm.com/docs/git-send-email#Documentation/git-send-email.txt---compose